docker/datahub-ingestion-base/base-requirements.txt

```diff
@@ -1,16 +1,17 @@
 # Generated requirements file. Run ./regenerate-base-requirements.sh to regenerate.
```

Comment:
Please don't make changes in this file. This file is updated separately. This is mainly a caching mechanism for our builds.